benchmarks/benchmarks.c

```diff
@@ -5,8 +5,9 @@
  *
  * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
  */
-
+#define _GNU_SOURCE
```

Comment:
This might create problems when trying to run the benchmark on non-gnu systems such as Mac OS X. Can we make the optimization optional?